### DISCUSSION TITLE
How do you disable user creation

### DISCUSSION THREAD

**Discussion by MLHComputer** [dani-garcia/vaultwarden#2901] [General]
i am trying to disable user creation i have SIGNUPS_ALLOWED=false but it still lets you create accounts

**Comment by BlackDex** [comment]
Have you actually tried to create an account??
You possibly get an error message if you try.

The button on the home screen doesn't disappear.

**Reply by MLHComputer** [reply]
i found that out i tried to remove this but it wont let me im sorry
